server/issue.go

```diff
@@ -73,6 +75,37 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 		create.Fields.Description = permalink
 	}
 
+	rootId := create.PostId
+	parentId := ""
+	if post.ParentId != "" {
+		// the original post was a reply
+		rootId = post.RootId
+		parentId = create.PostId
+	}
+
+	if len(create.RequiredFieldsNotCovered) > 0 {
+
+		v, _ := query.Values(create.Fields)
+
+		message := "The project you tried to create an issue for has **required fields** this plugin does not yet support. "
+		url := fmt.Sprintf("%v/secure/CreateIssueDetails!init.jspa", ji.GetURL())
+		query := "pid=10000&issuetype=10004"
```

Comment:
hardcoded pid and issuetype, but will be sent from client.  I was really hoping to borrow this google/go-querystring module which will fill a query string from a struct, but there are issues with capitalization and other slight changes that will need to be handled.